### PR TITLE
legit: Add version 1.2.0

### PR DESCRIPTION
**Pull request by TheRandomLabs**
[Legit](https://frostming.github.io/legit/) is a complementary command-line interface for Git optimized for workflow simplicity.

### CODE REVIEW

File: bucket/legit.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "description": "Complementary command-line interface for Git.",
+    "homepage": "https://frostming.github.io/legit/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/frostming/legit/blob/master/LICENSE"
+    },
+    "depends": "git",
+    "url": "https://github.com/frostming/legit/releases/download/1.2.0/legit.exe",
+    "hash": "2c4aa8a777141cad9171a7aec9b91d6570443644f7ca040903773620074d361b",
+    "post_install": "'y' | legit --install | Out-Null",
```
Comment (r15ch13) [inline bucket/legit.json:12]:
```suggestion
    "installer": {
        "script": "'y' | legit --install | Out-Null"
    },
```

File: bucket/legit.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        "script": "legit --uninstall | Out-Null"
+    },
+    "bin": "legit.exe",
+    "checkver": {
+        "github": "https://github.com/frostming/legit"
+    },
+    "autoupdate": {
+        "url": "https://github.com/frostming/legit/releases/download/$version/legit.exe",
+        "hash": {
+            "url": "$baseurl/legit.exe.sha256"
```
Comment (r15ch13) [inline bucket/legit.json:23]:
```suggestion
            "url": "$baseurl/$basename.sha256"
```

File: bucket/legit.json (hunk 1 above)
Comment (r15ch13) [reply bucket/legit.json:12]: Hm, this doesn't seem to work:
image

Comment (TheRandomLabs) [reply bucket/legit.json:12]: Interesting, it works for me:
image
Is there a better way to pass in that input?

Comment (r15ch13) [reply bucket/legit.json:12]: Do you use bash or pwsh/powershell? :smile:

Comment (TheRandomLabs) [reply bucket/legit.json:12]: PowerShell. :)

Comment (r15ch13) [reply bucket/legit.json:12]: Okay, it works fine in Windows Sandbox. (PS 5 and 7)

image

**Pull request merged**